File: 2_course/ЯПВУ/Laptev_AV1/Лаптев Александр/LaptevAV-2_2.go
package main
import ("fmt"
        "math"
)

func main() {
    var a float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&a)
    var b float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&b)
    var c float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&c)
    d := b * b - 4 * a * c
    if d < 0 {
        fmt.Print ("Korneyi net")
    }  else if d == 0 {
        fmt.Print ("x = ", -b / (2 * a))
    } else {
        fmt.Print ("x1 = ", (-b + math.Pow(d, 0.5)) / (2 * a), " x2 = ", (-b - math.Pow(d, 0.5)) / (2 * a))
    }
}

File: 2_course/ЯПВУ/Laptev_AV1/Лаптев Александр/LaptevAV-2_2.rb
# encoding utf-8
puts ('Vvedite veshestvennoe chislo: ')
a = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
b = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
c = gets.to_i
d = b * b - 4 * a * c
if (d < 0)
    puts ('Korney net')
elsif (d == 0)
    x = -b / (2 * a)
    puts ("#{x}")
else
    x1 = (-b + d ** 0.5) / (2 * a)
    x2 = (-b - d ** 0.5) / (2 * a)
    puts ("#{x1} #{x2}")
end